spec/spec_helper.rb

```diff
@@ -151,11 +151,14 @@ def main_app
   Valkyrie::Persistence::Memory::MetadataAdapter.new, :test_adapter
 )
 
-Valkyrie::MetadataAdapter
-  .find(:test_adapter)
-  .query_service
-  .custom_queries
-  .register_query_handler(Hyrax::CustomQueries::FindAccessControl)
+[Hyrax::CustomQueries::FindAccessControl,
+ Hyrax::CustomQueries::FindManyByAlternateIds].each do |handler|
+  Valkyrie::MetadataAdapter
+    .find(:test_adapter)
+    .query_service
```

Comment:
I prefer the approach in PR #4084 as you only have to find the test_adapter once.  Probably not a huge cost but why incur it if not needed.

https://github.com/samvera/hyrax/blob/a5ab3ca30ca078c8c4ced3da56f5ff6c9b9922d9/spec/spec_helper.rb#L155-L158

Reply:
happy to make the change.

Reply:
how does the new version look?